linchpin/provision/roles/azure/tasks/provision_azure_api.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Use Azure Api to manage resources
+  azure_rm_resource:
+    client_id: "{{ auth_var['client_id'] | default(omit) }}"
+    tenant: "{{ auth_var['tenant'] | default(omit) }}"
+    secret: "{{ auth_var['secret'] | default(omit) }}"
+    subscription_id: "{{ auth_var['subscription_id'] | default(omit) }}"
+    resource_group: "{{res_def['resource_group']|default(omit)}}"
+    provider: "compute"
+    api_version: "{{res_def['api_version']|default('2019-03-01')}}"
+    url: "{{ res_def['url']|default(omit)}}"
+    body: "{{ lookup('file', '{{workspace}}/body.json') }}"
```

Comment:
shouldn't this be `"{{ res_def['body'] }}"`?

Reply:
No, that's really long

Reply:
Then it has to be documented, and remove the body parameter from the schema. I also would recommend allowing the user to decide the file name - what if the user will have number azure_api resources and require different files?

Reply:
Good point